chapters/descriptorsets.txt

```diff
@@ -2070,24 +2070,14 @@ ifdef::VK_EXT_inline_uniform_block[]
 endif::VK_EXT_inline_uniform_block[]
 
 ifdef::VULKAN_1_1,VK_KHR_sampler_ycbcr_conversion[]
-Each combined image sampler descriptor for a multi-planar format may:
-consume multiple descriptors from the pool, as indicated by
-slink:VkSamplerYcbcrConversionImageFormatProperties::pname:maxCombinedImageSamplerDescriptorCount.
-The pname:descriptorCount for pools that will contain such descriptors must:
-be expanded to allow for this.
-
 [NOTE]
 .Note
 ====
-For example, consider a descriptor set layout binding with two descriptors
-and immutable samplers for multi-planar formats that have
-sname:VkSamplerYcbcrConversionImageFormatProperties::pname:combinedImageSamplerDescriptorCount
-values of `2` and `3` respectively.
-There are two descriptors in the binding and the maximum
-pname:combinedImageSamplerDescriptorCount is `3`, so descriptor sets with
-this layout consume `6` descriptors from the descriptor pool.
-To create a descriptor pool that allows allocating four descriptor sets with
-this layout, pname:descriptorCount must be at least `24`.
+When creating pools that will contain descriptor for combined image sampler
+for a multi-planar format, an application needs to account for non-trivial
+descriptor amount consumption when choosing the pname:descriptorCount value.
+This relates to pname:combinedImageSamplerDescriptorCount, and is specified
```

Comment:
For long-distance references we usually use slink:\<struct name\>::pname:\<field name\>. I think you could just append this to the previous sentence:

> ... choosing the pname:descriptorCount value, as indicated by slink:VkSamplerYcbcrConversionImageFormatProperties::pname:combinedImageSamplerCount.